session/session.go

```diff
@@ -3204,11 +3133,8 @@ func (s *session) PrepareTSFuture(ctx context.Context, future oracle.Future, sco
 	return nil
 }
 
-func (s *session) GetPreparedTSFuture() oracle.Future {
-	if future := s.txn.txnFuture; future != nil {
-		return future.future
-	}
-	return nil
+func (s *session) GetPreparedTxnFuture() sessionctx.TxnFuture {
```

Comment:
`GetPreparedTxnFuture` should return nil when the txn is not prepared.

Reply:
Done.